src/AdsImporter/Database/MongoMigration.php

```diff
@@ -250,12 +250,31 @@ public function addOrUpdateNode(Node $node): void
             'port' => $node->getPort(),
             'publicKey' => $node->getPublicKey(),
             'status' => $node->getStatus(),
+            'version' => $node->getVersion(),
 
         ];
 
         $this->nodeCollection->update(['_id' => $node->getId()], $document, $this->mongoUpdateOptions);
     }
 
+    /**
+     * @param string $nodeId
+     * @return null|string
+     */
+    public function getNodeVersion(string $nodeId): ?string
+    {
+        $cursor = $this->transactionCollection->find([
+            'type' => 'connection',
+            'nodeId' => $nodeId,
+        ])->sort(['_id' => -1])->limit(1);
```

Comment:
`findOne` ?

Reply:
I just followed the convention https://github.com/adshares/ads-operator/blob/master/src/AdsImporter/Database/MongoMigration.php#L292

Reply:
Ok. I should have used `findOne` also there :)